src/components/layout/PageWrapper.js

```diff
@@ -18,12 +18,12 @@ const useStyles = makeStyles((theme) => ({
 function PageWrapper(props) {
     const { appBarHeight } = props;
     const classes = useStyles();
-
+    const theme = useTheme();
     return (
         <div
             className={classes.wrapper}
             style={{
-                maxHeight: `calc(100vh - ${appBarHeight}px`,
+                maxHeight: `calc(100vh - ${appBarHeight + theme.spacing(1)}px)`,
             }}
         >
             {props.children}
```

Comment:
I added the extra spacing so that the bottom half of the Paper containing the pagination items is not cut-off.

Reply:
May be I didn't phrase it will. Checkout below screenshots. 
 QA - See the bottom of pagination bar:
![image](https://user-images.githubusercontent.com/1250790/83641444-2b341880-a5cb-11ea-9038-d4eeb03ddffe.png)

After fix - 

![image](https://user-images.githubusercontent.com/1250790/83641504-42730600-a5cb-11ea-955a-051fafba914d.png)